.tools/test/stacks/config/variables.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Why not make this an array of key/value pairs so we don't have to iterate over it later?